devel/gitlab-runner/Portfile

```diff
@@ -29,14 +29,18 @@ checksums           rmd160  9f61ee71f3578a182ef17751ae458a1371ab521d \
 # Allow Go to fetch deps at build time
 build.env-delete    GO111MODULE=off GOPROXY=off
 
-# Reproduce the "build_simple" target from the upstream Makefile
+# Reproduce the "runner-bin-host" target from the upstream Makefile
 # Lookup REVISION from https://gitlab.com/gitlab-org/gitlab-runner/-/releases/v${version}
+set release_info [exec curl -s "https://gitlab.com/api/v4/projects/gitlab-org%2Fgitlab-runner/releases/v${version}"]
```

Comment:
❗ This is a bad idea.

If you do this, then every single time this Portfile is even **_parsed_**, MacPorts will have to do a `curl` exec.

This can happen when MacPorts is doing a routine `portindex`, someone does a `port info gitlab-runner`, etc.

We should stick to hardcoding the `REVISION` and `BRANCH` id for now.  What you propose here will have too many negative impacts on workflows that may not even have anything to do with `gitlab-runner`.

Reply:
The main reason behind this was avoid errors like in https://github.com/macports/macports-ports/commit/5a7f5d10128e19e4f8857237d8eab77ef395d7f8.

Reply:
Is it better to calculate that at build phase?